migtests/scripts/run-test.sh

```diff
@@ -65,17 +65,22 @@ main() {
 	tail -20 ${EXPORT_DIR}/reports/report.txt
 
 	step "Export data."
-	export_data
-	ls -l ${EXPORT_DIR}/data
+	{ #TRY
+		export_data 
+		export_data_exit_code=$?
+	}|| { #CATCH
+		# Print debezium.log file if present and exportDataDone flag file is not present
+		if [ -f "${EXPORT_DIR}/logs/debezium.log" ] && [ ! -f "${EXPORT_DIR}/metainfo/flags/exportDataDone" ]
+		then
+			echo "Printing debezium.log file"
+			tail -n 100 ${EXPORT_DIR}/logs/debezium.log
+		else
+			echo "No debezium.log found."
+		fi	
+		exit 1 #since the TRY block failed
+	}
 
-	# Print debezium.log file if present and exportDataDone flag file is not present
-	if [ -f "${EXPORT_DIR}/logs/debezium.log" ] && [ ! -f "${EXPORT_DIR}/metainfo/flags/exportDataDone" ]
```

Comment:
Is this condition not working currently? Even if exit code is 0, this condition should be met, right?

Reply:
I'm assuming this works currently. this PR is not about fixing the print of log file, but more about making run_tests.sh exit when export data fails. 

Correct?

Reply:
currently if actually export data fails with exit code=1, then this won't be printed, because the `run-tests.sh` script will exit because of `set -e`